packages/cli/src/dbt/targets/Bigquery/index.ts

```diff
@@ -24,13 +25,17 @@ type BigqueryTarget = {
 
 export const bigqueryTargetJsonSchema: JSONSchemaType<BigqueryTarget> = {
     type: 'object',
+    required: ['project'],
```

Comment:
is this essential for the PR?

Reply:
This already existed but I removed `dataset` from there, since it will now either `schema` or `dataset` are required in `oneOf`. I must've moved this up in the schema somehow, pushed a commit to put it back in place 👌